src/components/Footer.js

```diff
@@ -1,8 +1,116 @@
-import React from "react";
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faLinkedin, faGithub } from '@fortawesome/free-brands-svg-icons';
 
 const Footer = () => {
+  const currentYear = () => {
+    const today = new Date();
+    return today.getFullYear();
+  };
+
   return (
-    <h1> Hello This is Footer component </h1>
+    <footer className="bg-primary text-white pt-3">
+      <div className="d-flex justify-content-around">
+        <section>
+          <h6 className="text-uppercase">Social</h6>
+          <ul className="list-unstyled">
+            <li>
+              <a
+                className="footer-link text-white"
+                href="https://github.com/CodeYourFuture/project-quiz/"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                <FontAwesomeIcon
+                  className="mr-2"
+                  aria-hidden="true"
+                  aria-label="Github icon"
```

Comment:
Since we are hiding this icon from screen readers, we no longer need `aria-label`

Reply:
Oh, my bad I forgot to remove this